lib/chef/resource/ohai.rb

```diff
@@ -27,8 +27,12 @@ class Ohai < Chef::Resource
 
       description "Use the ohai resource to reload the Ohai configuration on a node. This allows recipes that change system attributes (like a recipe that adds a user) to refer to those attributes later on during the chef-client run."
 
-      property :ohai_name, name_property: true, introduced: "12.17"
-      property :plugin, String
+      property :ohai_name,
+               deprecated: true,
+               name_property: true, introduced: "12.17"
```

Comment:
yeah but that's the name property, shouldn't we do the apt_update trick to allow it to be unnamed?

Reply:
The name_property isn't actually used. It's just a dangling property that makes no sense in the context of the way the resource works.

Reply:
yeah i see that `apt_update` doesn't actually  use a name_property either...  carry on...